packages/web-react/src/components/Item/Item.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { SpiritItemProps } from '../../types';
+import { useStyleProps } from '../../hooks';
+import { HelperText } from '../Field';
+import { Icon } from '../Icon';
+import { useItemStyleProps } from './useItemStyleProps';
+
+const Item = <T extends ElementType = 'div'>(props: SpiritItemProps<T>): JSX.Element => {
+  const { label, elementType: ElementTag = 'div', iconName, helperText, isSelected, ...restProps } = props;
+  const { classProps, props: modifiedProps } = useItemStyleProps({ isSelected, ...restProps });
+  const { styleProps, props: otherProps } = useStyleProps(modifiedProps);
+
+  return (
+    <ElementTag {...otherProps} {...styleProps} className={classNames(classProps.root, styleProps.className)}>
+      {iconName && (
+        <span className={classNames(classProps.icon.root, classProps.icon.start)}>
+          <Icon name={iconName} />
+        </span>
+      )}
+      <span className={classProps.label}>{label}</span>
```

Comment:
Wouldn't it be better to use the `children` prop for this case? I can imagine that we will use the Item component as a wrapper in the future and this will be blocking, IMHO. What do you mean?

Reply:
As we agreed it remain as it is for now but it now accept ReactNode elements too.